src/ethereum_test_base_types/composite_types.py

```diff
@@ -462,3 +462,29 @@ class AccessList(CamelModel):
     def to_list(self) -> List[Address | List[Hash]]:
         """Return access list as a list of serializable elements."""
         return [self.address, self.storage_keys]
+
+
+class ForkBlobSchedule(CamelModel):
+    """Representation of the blob schedule of a given fork."""
```

Comment:
is this applied retrospectively to cancun tests?

Reply:
Yes.

Reply:
Here's an example included in the test fixtures of this PR:
https://github.com/ethereum/execution-spec-tests/blob/c73f68596c8a47a2370ddc2f9e6c74e56d8327ce/src/ethereum_test_specs/tests/fixtures/chainid_cancun_blockchain_test.json#L131-L136